content/docs/overview/tracers.md

```diff
@@ -1,3 +1,63 @@
 ---
 title: Tracers
 ---
+
+OpenTracing provides an open, vendor-neutral standard for distributed tracing systems instrumentation, providing an interface to the different distributed tracers. This means that instrumentation would remain the same irrespective of the tracer system being used by the developer. In order to instrument the web application using OpenTracing specification, a compatible OpenTracing tracer must be deployed and listening for incoming span requests. A list of the all the OpenTracing tracers is available [here](./../../supported-tracers). The following section discusses Global Tracers and the various tracer properties that must be taken into account while choosing one.
+
+<br>
+
+## Choosing a Tracer
```

Comment:
I am not sure this is the right way to phrase the question. Tracer is just an extension of the tracing backend, one cannot choose them independently. And picking the backend purely based on the features of the tracer (instrumentation library) is not useful. 

I think this page can be simply a list of tracing systems that supply OT-compatible tracers. Performing a feature comparison of those tracing systems is not in scope of the documentation, since it's going to become out of date quickly. For example:

The following tracing backends have OpenTracing-compatible tracing libraries:

| Tracing System | Go tracer | Java tracer  | 
| ------------- |-------------| -----|
| name      | [link](link) | n/a |

Given that we probably have more languages that tracing systems, may want to transpose the table. Or not do a table, since it's going to be hard to edit, just have a list of tracing backends with sublists of clients by language.